bcda/models/models_test.go

```diff
@@ -277,3 +278,28 @@ func (s *ModelsTestSuite) TestGetBeneficiaryIDs() {
 	assert.Equal(100, len(beneficiaryIDs))
 
 }
+
+func (s *ModelsTestSuite) TestGroupStructs() {
```

Comment:
The essential things the testing needs to do: 
(a) unmarshal a json string into the structs (as it will do in an api handler)
(b) save the json to the db
(c) get the json out of the db into structs
(d) make sure the input of a matches the output of (c) (marshal the output from (c) and compare strings with (a)'s input?) 
I see (b) and (c), but not (a) and (d)